cmd/jaeger/internal/extension/remotesampling/config_test.go

```diff
@@ -51,6 +51,13 @@ func Test_Validate(t *testing.T) {
 			},
 			expectedErr: "File.Path: non zero value required",
 		},
+		{
+			name: "File provider can have empty file path",
+			config: &Config{
+				File: &FileConfig{Path: "", ReloadInterval: -1},
+			},
+			expectedErr: "must be a positive value",
+		},
```

Comment:
shouldn't the name of this test be something like `file provider has negative reload interval`?

Reply:
+1